daos-stack/daos
# DAOS-10854 test: verify multi-user dfuse permissions

Test-tag: dfuse_mu
Skip-unit-tests: true
Skip-fault-injection-test: true
   
- Add ExecutableCommand.run_user
  - Used to run a command as another user
  - Update dfuse to run commands as another user
- Add ExecutableCommand.bind_cores
  - Used to run a command with taskset
  - Update dfuse to use this option
- Add utility verify_perms.py to verify owner, group, other permissions
  on a file or directory.
  - Add VerifyPermsCommand wrapper to run remotely
- Add dfuse/mu_perms.py to verify multi-user permissions
- Adjust dfuse/mu_mount.py to use updated test utilities
    
Required-githooks: true
    
Signed-off-by: Dalton Bohning <dalton.bohning@intel.com>

### Before requesting gatekeeper:


### Gatekeeper:
